examples/test-local-repo-digest-ts/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as docker from "@pulumi/docker";
+
+const localImg = new docker.Image("my-image", {
+    imageName: "pulumi-bot/local-repo-digest",
+    skipPush:true,
+    build: {
+        platform: "linux/amd64",
+    }
+});
+
+export const imageName = localImg.imageName;
+
+export const repoDigest = localImg.repoDigest
```

Comment:
It would be really nice to also have a more complete E2E example that uses `docker.Container` to run the image, and then to have an EditDirs update test that changes the build to show that this triggers the Container to restart with the new image.  This is the local equivalent of the standard scenario for remote container registries (and is the thing that will prove that this does fix https://github.com/pulumi/pulumi-docker/issues/633).

The test could validate that the second update replaces the `Container` and/or that an HTTP request to the container returns the expected new value based on the change.

Reply:
Hmmmm I like this. Will remove the fix for #633 and add a test that then closes that in a follow-up.